data/saas/config/saas_example_config.yml

```diff
@@ -38,10 +41,8 @@ saas_config:
                 - dataset: saas_connector_example
                   field: conversations.id
                   direction: from
+          data_path: conversation_messages
```

Comment:
Unrelated to our changes but I cleaned this up since I was here to add some new collections for testing.